### PR TITLE
XSOAR RF 2.2 release

### PR DESCRIPTION
**Pull request by rderkachrf**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR request address two things:
- Related entities section in the war room output is now in table format
- Added one more command `!recordedfuture-links` that brings a new Recorded Future dataset
- Fixed Playbooks
## Screenshots
image
image

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/RecordedFuture/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "RecordedFuture v2",
     "description": "Recorded Future App v2",
     "support": "partner",
-    "currentVersion": "1.1.3",
+    "currentVersion": "2.2.0",
```
Comment (ChanochShayner) [inline Packs/RecordedFuture/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.1.4",
```
The version jump can be confusing for customers.

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+#### Integrations
+##### Recorded Future v2
+- Added one more command `!recordedfuture-links` that brings a new Recorded Future dataset
+- Related entities section in the war room output is now in table format
+
+#### Playbooks
+##### Recorded Future URL Reputation
+- fixed the `recordedfutureriskrules` field name that is used by SetIndicator task
```
Comment (ChanochShayner) [inline Packs/RecordedFuture/ReleaseNotes/2_2_0.md:9]:
```suggestion
- Fixed the `recordedfutureriskrules` field name that is used by SetIndicator task.
```

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+##### Recorded Future CVE Reputation
+- fixed the `recordedfutureriskrules` field name that is used by SetIndicator task
```
Comment (ChanochShayner) [inline Packs/RecordedFuture/ReleaseNotes/2_2_0.md:11]:
```suggestion
- Fixed the `recordedfutureriskrules` field name that is used by SetIndicator task.
```

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+#### Integrations
+##### Recorded Future v2
+- Added one more command `!recordedfuture-links` that brings a new Recorded Future dataset
```
Comment (ChanochShayner) [inline Packs/RecordedFuture/ReleaseNotes/2_2_0.md:4]:
```suggestion
- Added `!recordedfuture-links` command that brings a new Recorded Future dataset.
```

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- Related entities section in the war room output is now in table format
```
Comment (ChanochShayner) [inline Packs/RecordedFuture/ReleaseNotes/2_2_0.md:5]:
```suggestion
- Related entities section in the war room output is now in table format.
```

**Review by ChanochShayner — COMMENTED** [review]
@rderkachrf Looks good!
Please see my comments.

File: Packs/RecordedFuture/pack_metadata.json (hunk 1 above)
Comment (rderkachrf) [reply Packs/RecordedFuture/pack_metadata.json:5]: So it should not be the same as we have in RecordedFuture.py ?
https://github.com/demisto/content/pull/14140/files#diff-70a479bb2ddd21051961ac8828ca0b2f166da1bc512bf9de49aa21a1715c8765L17-R17

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md (hunk 2 above)
Comment (rderkachrf) [reply Packs/RecordedFuture/ReleaseNotes/2_2_0.md]: fixed

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md (hunk 3 above)
Comment (rderkachrf) [reply Packs/RecordedFuture/ReleaseNotes/2_2_0.md]: fixed

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md (hunk 4 above)
Comment (rderkachrf) [reply Packs/RecordedFuture/ReleaseNotes/2_2_0.md]: fixed

File: Packs/RecordedFuture/ReleaseNotes/2_2_0.md (hunk 5 above)
Comment (rderkachrf) [reply Packs/RecordedFuture/ReleaseNotes/2_2_0.md]: fixed

**Comment by ChanochShayner** [comment]
@rderkachrf looks great!
Do you want a short demo to be set to see that the modified commands look fine?
Or you can send a short video of running the modified commands.

**Comment by rderkachrf** [comment]
> @rderkachrf looks great!
> Do you want a short demo to be set to see that the modified commands look fine?
> Or you can send a short video of running the modified commands.

We have already scheduled a demo today at  2:00 pm CET

**Comment by ChanochShayner** [comment]
> > @rderkachrf looks great!
> > Do you want a short demo to be set to see that the modified commands look fine?
> > Or you can send a short video of running the modified commands.
> 
> We have already scheduled a demo today at 2:00 pm CET

With who?

**Review by ChanochShayner — APPROVED** [review]

**Pull request merged**